lib/query/query_supporters.rb

```diff
@@ -231,6 +231,7 @@ def self.full_filter_expr(np_id, query)
     if query[:search].present?
       expr = expr.and_where(%Q(
         supporters.fts @@ websearch_to_tsquery('english', $search)
+        OR supporters.name % $search
```

Comment:
Are we sure this leads to useful results? Or do the results become less useful?

Reply:
I edited the description with some use cases. I think in general users can distinguish when a result is not what they want, but there's a scenario that we might want to look at:

We have more than 1 Cacau (different people) who donated. One of them has the name "Cacau" in our records, and the other one has "Cacau Borges". I want to find the records from "Cacau Borges", but the results from "Cacau" are going to come up. The emails are going to help them to distinguish the results. A similar behavior already happens if you look for the first name only: you won't find exclusively supporters that match exactly "Cacau", you'll also find "Cacau Borges".

Reply:
So, if you search for Cacau Borges, you get "Cacau"? Does it put the records with Cacau Borges first?

Reply:
Not necessarily because right now we're ordering by `last_payment` date. What do you think?

Reply:
There's a way to order by the name similarity.

Reply:
I think it would be helpful to have @wendylbolm's viewpoint here.